frontend/src/app/features/site/Search.css

```diff
@@ -265,3 +265,10 @@ div.search-parent {
   padding: var(--layout-padding-medium) var(--layout-padding-small)
     var(--layout-padding-medium) var(--layout-padding-small);
 }
+
+
+.disabled-btn
+{
+  color: var(--surface-color-primary-disabled, #EDEBE9);
```

Comment:
I don think this variable is defined in design token. Can you pls check at your end and use a suitable one?

Reply:
updated with right one.